src/test/resources/de.digiwill/editProfile.feature

```diff
@@ -0,0 +1,79 @@
+@uitest
+Feature: Edit Profile
+  As a mortal
+  I want to edit my profile information
+
+  Background:
+    Given "/?login" is open
+    Given A user with email "example@mail.com" and password "password" exists
+    When Enter Email "example@mail.com", password "password" and login
+    Then Login "succeeds"
+
+  Scenario Outline: 01 - Edit Personal Data successful
+    When I click on element with id "userButtonHeader"
+    And I click on element with id "editProfileButton"
+    And I clear field with id "firstNameInput"
+    And I enter "<firstName>" into field with id "firstNameInput"
+    And I clear field with id "surNameInput"
+    And I enter "<surName>" into field with id "surNameInput"
+    And I clear field with id "birthday"
+    And I enter "<birthday>" into field with id "birthday"
+    And I enter "<streetAddress>" into field with id "streetAddressInput"
+    And I enter "<zipCode>" into field with id "zipCodeInput"
+    And I enter "<city>" into field with id "cityInput"
+    And I enter "<country>" into field with id "countryInput"
+    And I click on element with id "saveProfileButton"
+    Then I'm on page with title "DigiWill - Your Profile"
```

Comment:
Shouldn't we check, that the information actually got updated?

Reply:
This makes sense. Do you have a suggestion on how we can test this?